circuits/cpp/src/aztec3/circuits/recursion/play.test.cpp

```diff
@@ -1,117 +0,0 @@
-#include "index.hpp"
-
-#include <barretenberg/barretenberg.hpp>
-
-#include <gtest/gtest.h>
-
-namespace aztec3::circuits::recursion {
-
-using namespace aztec3::utils::types;
-
-class play_tests : public ::testing::Test {
-  protected:
-    static void SetUpTestSuite() { barretenberg::srs::init_crs_factory("../barretenberg/cpp/srs_db/ignition"); }
-};
-
-TEST_F(play_tests, circuit_play_app_circuit)
```

Comment:
These were tests that Mike wrote when he first started playing around with az3 circuits. They were outdated and no longer of value.